src/common/components/select-repositories-page/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchUserRepositories } from '../../actions/repositories';
+import SelectRepositoryList from '../select-repository-list';
+
+class SelectRepositoriesPage extends Component {
+  componentDidMount() {
+    const { authenticated } = this.props.auth;
+
+    if (authenticated) {
+      this.props.dispatch(fetchUserRepositories());
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <h2>You’ve successfully imported all your repos from GitHub!</h2>
```

Comment:
This heading looks bit scary.
Listing user repositories doesn't mean we did anything with them, especially we didn't 'import' them or anything.